### PR TITLE
EWPP-811: Add optional heading field to Timeline paragraph.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

File: modules/oe_paragraphs_timeline/oe_paragraphs_timeline.post_update.php
Comment (upchuk) [inline modules/oe_paragraphs_timeline/oe_paragraphs_timeline.post_update.php:23]: Don't you know the entity type?

File: modules/oe_paragraphs_timeline/oe_paragraphs_timeline.post_update.php
Comment (upchuk) [inline modules/oe_paragraphs_timeline/oe_paragraphs_timeline.post_update.php:25]: Let's also check if a field called "oe_title" doesnt exist already.

**Review by upchuk — CHANGES_REQUESTED** [review]

**Review by upchuk — CHANGES_REQUESTED** [review]
Accepting but keeping the PR blocked until the epic is ready to be merged.

**Review by sergepavle — APPROVED** [review]

**Review by upchuk — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]

**Pull request merged**
